test/module-index.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const path = require('path');
+const { filter, find } = require('lodash');
+const { expect } = require('chai');
+const glob = require('glob').sync;
+const { fromUnicodeToHexcode, stripHexcode } = require('emojibase');
+
+const argv = require('optimist').demand('openmoji-data-json').argv;
+const openmojiDataJson = argv['openmoji-data-json'];
+const openmojis = require(openmojiDataJson);
+
+const openmoji = require('../index');
+
+describe('Data integrity of index.js exports', () => {
+
+  describe('The paths in openmojis', () => {
```

Comment:
We try normally to write it as a question e.g. "Paths to SVG production files existing?" see the other tests.

Reply:
I see. I tried to follow the intention of mocha's `describe` and `it` methods. The `describe` block presents a module, feature or a subfeature and `it` blocks define how the feature should behave. For example see https://mochajs.org/#getting-started

However, I think it is ok to describe a question. Feels a bit unusual but I guess it works well in the output. I will convert the strings into questions.

Reply:
Yes + thanks. But it is helpful as we show the same "question" in the testing results in https://openmoji-tester.glitch.me/ ... and the realized that this is more readable for all audiences.